app/static/src/app/serialise.ts

```diff
@@ -123,4 +125,13 @@ export const deserialiseState = (targetState: AppState, serialised: SerialisedAp
         ...targetState,
         ...serialised
     });
+
+    // Initialise selected variables if required
+    const { model } = targetState;
+    if (model.odinModelResponse?.metadata?.variables && !model.selectedVariables?.length
+        && !model.unselectedVariables?.length) {
+        /* eslint-disable no-param-reassign */
```

Comment:
Not sure why you need this.

Reply:
By default eslint frowns on modifying parameters to functions, but in this case we do want to modify the object that we're serialising into (while retaining some of its default state).